src/apiProxy/api-proxy.js

```diff
@@ -5,6 +5,16 @@ const externalRequest$ = Rx.Observable.bindNodeCallback(externalRequest);
 import * as apiConfigCreators from './apiConfigCreators';
 import { duotoneRef } from '../util/duotone';
 
+const parseResponseFlags = ({ headers }) =>
+	(headers['x-meetup-flags'] || '')
+		.split(',')
+		.filter(pair => pair)
```

Comment:
Yeah, since an empty string will yield `['']` when using `split` and if there was a dangling comma you would get an empty string from `split`, this just 'validates' the input to only pass through populated values. I can leave a comment so that maintainers don't accidentally remove it